### PR TITLE
created function that pulls all required info from from database

### PR DESCRIPTION
**Pull request by MotNosniktaw**

### CODE REVIEW

**Comment by RachmannJoubert** [comment]
I'm don't think this query will work.

**Comment by RachmannJoubert** [comment]
All looks good.

File: index.php [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
 );
 
 $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
+/**
+ *
+ * Returns all desired item values for all items.
+ * Where many images are stored for an item, only the first added (MAIN image) is returned
+ *
+ * @param $database
```
Comment (sjr-mayden) [inline index.php:17]: What is the database? Should mention it's the connection established using PDO or similar

File: index.php [hunk 2]
```diff
@@ -1,10 +1,41 @@
...
+ * @return array
```
Comment (sjr-mayden) [inline index.php:18]: Context-wise what's returned here? Other devs can see it's an array from the type hinting done on line 20

File: index.php [hunk 3]
```diff
@@ -1,10 +1,41 @@
...
+            LEFT JOIN `countries`
+                ON `countries`.`id` = `guitars`.`countryID`
+            GROUP BY `guitars`.`id`;'
+    );
+
+    $sqlGetAll->execute();
+    return $sqlGetAll->fetchAll();
+}
+
+$returnAll = getAllFromDatabase($db);
```
Comment (sjr-mayden) [inline index.php:40]: Naming of $returnAll isn't semantic, $allGuitars instead, maybe?

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (MotNosniktaw) [reply index.php:17]: have added more relevant information to DocBlock that details what input parameters and outputs should be

File: index.php (hunk 2 above)
Comment (MotNosniktaw) [reply index.php:18]: have added more relevant information to DocBlock that details what input parameters and outputs should be

File: index.php (hunk 3 above)
Comment (MotNosniktaw) [reply index.php:40]: have changed variable to $allGuitars

**Pull request merged**